scopes/component/forking/fork.cmd.ts

```diff
@@ -9,22 +9,38 @@ export type ForkOptions = {
 };
 
 export class ForkCmd implements Command {
-  name = 'fork <source-id> [target-name]';
-  description = 'EXPERIMENTAL. create a new component out of an existing one';
+  name = 'fork <source-component-id> [target-component-name]';
+  description = 'EXPERIMENTAL. Creates a new component out of an existing one (copies source files and config).';
+  arguments = [
+    { name: 'source-component-id', description: 'the component id of the source component' },
+    {
+      name: 'target-component-name',
+      description: 'the name for the new component (component name without scope)',
+    },
+  ];
+
   extendedDescription = `note that [target-name] is the name only without the scope.
```

Comment:
```
- [target-name]
+ [target-component-name]
```

Reply:
removed the extended description and added the --scope flag comment in the target-component-name arg description